src/Assets/Asset.php

```diff
@@ -1031,6 +1032,14 @@ public function getQueryableValue(string $field)
         return $field->fieldtype()->toQueryableValue($value);
     }
 
+    private function getDirtyArray()
```

Comment:
Can this be made `public`, like it is in `Entry` and `User`?

Reply:
Done